store/json.go

```diff
@@ -197,7 +198,7 @@ func (kvs *jsonFileStore) Lock(timeout time.Duration) error {
 		return errors.Wrap(err, "processLock acquire error")
 	}
 
-	log.Printf("Acquired process lock with timeout value of %v ", timeout)
+	log.Logger.Info("Acquired process lock with value of ", zap.Duration("timeout", timeout))
```

Comment:
can it be like previous message "Acquired process lock with **timeout** value of"

Reply:
The message should just be `Acquired process lock`. The rest of the context is provided by the zap parameter.